docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -206,103 +208,151 @@ mode uses block devices to create the thin pool. The following procedure shows
 you how to configure a Docker host to use the `devicemapper` storage driver in
 a `direct-lvm` configuration.
 
-> **Caution:** If you have already run the Docker daemon on your Docker host
+> **Caution:** If you have already run the Engine daemon on your Docker host
 > and have images you want to keep, `push` them Docker Hub or your private
 > Docker Trusted Registry before attempting this procedure.
 
 The procedure below will create a 90GB data volume and 4GB metadata volume to
 use as backing for the storage pool. It assumes that you have a spare block
 device at `/dev/xvdf` with enough free space to complete the task. The device
```

Comment:
In the example, it would probably make sense to either update this to `/dev/xvdf` everywhere `/dev/sdd` is mentioned or replace this instance with `/dev/sdd` just for consistency sake.